web/packages/teleterm/src/ui/DocumentAuthorizeWebSession/DocumentAuthorizeWebSession.tsx

```diff
@@ -111,6 +114,32 @@ export function DocumentAuthorizeWebSession(props: {
               </Text>
             </Alert>
           )}
+          {!isRequestedUserLoggedIn && (
+            <Alert mb={0}>
+              <Text>
+                Requested user is not logged in.
+                <br />
+                You are logged in as <b>{rootCluster.loggedInUser?.name}</b>. To
+                authorize this web session request, please{' '}
+                <Link
+                  css={`
+                    cursor: pointer;
+                  `}
+                  onClick={() => {
+                    ctx.commandLauncher.executeCommand('cluster-logout', {
+                      clusterUri: rootCluster.uri,
+                    });
+                  }}
+                >
```

Comment:
This is not great, but `<ButtonText>` seems to be even worse. If we ever need to do something like this on master, it'd be a good idea to add some kind of `inline` prop to `<ButtonText>` that adds underline and removes font-size, color and min-height.

Reply:
Agree, I wasn't very happy doing it :(